src/proteomes/adapters/proteomesConverter.ts

```diff
@@ -0,0 +1,98 @@
+import { Lineage, Xref, Citation } from '../../shared/types/apiModel';
+
+export type Taxonomy = {
+  scientificName: string;
+  taxonId: number;
+  commonName: string;
+  mnemonic: string;
+  synonyms: string[];
+};
+
+export type GenomeAnnotation = {
+  source: string;
+  url?: string;
+};
+
+export type Component = {
+  name: string;
+  description: string;
+  genomeAnnotation: GenomeAnnotation;
+  proteomeCrossReferences: Xref[];
+  proteinCount: number; // used in the entry for each component TODO: eventually will be supported by backend in 2021_02 - 2021_03
+};
+
+export type CpdReport = {
+  proteomeCount: number;
+  stdCdss: number;
+  averageCdss: number;
+  confidence: number;
+  status: string;
+};
+
+export type BuscoReport = {
+  complete: number;
+  completeSingle: number;
+  completeDuplicated: number;
+  fragmented: number;
+  missing: number;
+  total: number;
+  lineageDb: string;
+  score: number;
+};
+
+export type ProteomeCompletenessReport = {
+  cpdReport: CpdReport;
+  buscoReport: BuscoReport;
```

Comment:
Are those (both cpd and busco) present on all proteome entries? I though some of them didn't have this as there is not always a busco visualisation on the current website

Reply:
* CPD always present (at least in current site data)
* BUSCO optional